cmd/sebak/cmd/genesis.go

```diff
@@ -57,23 +57,22 @@ func init() {
 // Params:
 //   address   = public address of the account owning the genesis block
 //   networkID = `--network-id` argument, used for the block's checkpoint
-//   balance   = Amount of coins to put in the account
-//               If not provided, `flagBalance`, which is the value set in the env
-//               when called from another module, will be used
-//   balance   = Amount of coins to put in the account
-//               If not provided, a default value will be used
+//   balanceStr = Amount of coins to put in the account
+//                If not provided, `flagBalance`, which is the value set in the env
+//                when called from another module, will be used
+//   storageConfigUri = URI to include storage path("file://path")
+//                      If not provided, a default value will be used
 //
 // Returns:
 //   If an error happened, returns a tuple of (string, error).
 //   The string argument represent the name of the flag which errored,
 //   and error is the more detailed error.
 //   Note that only one needs be non-`nil` for it to be considered an error.
 //
-func MakeGenesisBlock(addressStr, networkID, balanceStr, storage string) (string, error) {
+func MakeGenesisBlock(addressStr, networkID, balanceStr, storageConfigUri string) (string, error) {
```

Comment:
@Geod24 I renamed it to storageConfigUri cause `storageConfig` is duplicated with https://github.com/bosnet/sebak/pull/332/files#diff-c21723bcdd95c8506c590ab62f284135R111